### PR TITLE
Throttle rendering

### LINKED ISSUE #3: Improve interactive performance
Performance on mybinder is very poor. We need to improve the performance and set up render timers that will prevent overloads of frames trying to stream back to the client

### PR DESCRIPTION
**Pull request by banesullivan**
Resolve #3

### CODE REVIEW

**Comment by banesullivan** [comment]
Performance still isn't amazing on MyBinder. It appears the DOM events are the bottleneck now

I tried throttling the dom event locally but it didn't seem to work... will keep experimenting

**Comment by martinRenou** [comment]
> It appears the DOM events are the bottleneck now

It might be something to improve in `ipycanvas` directly, preventing too many mouse events to be sent to the Python kernel.

**Comment by banesullivan** [comment]
Where exactly could we do this? I tried overriding `Canvas._mouse_move_callbacks` with a throttle decorator but this had no effect. The only thing is we definitely do not want to throttle/limit any click events, only the mouse move events

**Comment by martinRenou** [comment]
This is something that should be done JavaScript side, you cannot have control on it Python side unfortunately

**Comment by banesullivan** [comment]
Ah, I see. I will merge these changes if @davidbrochart thinks they are okay and wait for the remainder to be fixed in `ipycanvas` or `ipyevents`

**Comment by martinRenou** [comment]
I missed that ipyevents was involved.

@mwcraig do you think it could make sense to add the ability for users to limit mouse events (from the JavaScript side) in ipyevents? Or is it already possible?

File: ipyvtk_simple/viewer.py [hunk 1]
```diff
@@ -115,11 +118,12 @@ def send_pending_mouse_move_event(self):
             self.interactor.MouseMoveEvent()
             self.last_mouse_move_event = None
 
+    @throttle(0.02)
```
Comment (davidbrochart) [inline ipyvtk_simple/viewer.py]: 20ms is really short, why don't you try with something like `@throttle(0.1)`?

**Comment by davidbrochart** [comment]
> I will merge these changes if @davidbrochart thinks they are okay

LGTM except for the very small values, I don't know if it has any effect.

**Comment by banesullivan** [comment]
Thanks, @davidbrochart!

**Pull request merged**